EIPS/eip-7733.md

```diff
@@ -0,0 +1,232 @@
+---
+eip: 7733
```

Comment:
```suggestion
eip: 7520
```
Assigning next sequential EIP/ERC number.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
Thank you so much. Already updated